source/ruby.txt

```diff
@@ -0,0 +1,34 @@
+.. _ruby-language-center:
+
+===================
+MongoDB Ruby Driver
+===================
+
+.. toctree::
+   :titlesonly:
+
+   Ruby Driver  <https://www.mongodb.com/docs/ruby-driver/current/>
+   Ruby Mongoid ODM  <https://www.mongodb.com/docs/mongoid/current/>
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Introduction
+------------
+
+Welcome to the documentation site for the official MongoDB Ruby 
+driver and Mongoid ODM. See the :ruby:`official Ruby driver
+documentation site <>` to learn how to begin using the driver.
+
+Object Document Mapper
+----------------------
+
+If your application uses validations, associations, and other high-level
+data modeling functions, then you may need to install an Object
```

Comment:
Suggestion:

I don't think the user needs to install it, but they could benefit from it.

```suggestion
You can use Mongoid, the official MongoDB Ruby ODM, to streamline your application's logic for validation, data model associations, and other high-level data modeling functions.
```